src/exec/mod.rs

```diff
@@ -27,6 +27,10 @@ use self::input::{basename, dirname, remove_extension};
 pub use self::job::{batch, job};
 use self::token::Token;
 
+/// When `ExecutionMode` is `Batch` and number of results is less
+/// than this value, results will be sorted
+const SORT_THRESHOLD: usize = 100;
```

Comment:
This would be similar to `MAX_BUFFER_LENGTH` from `src/internal.mod`. Could we reuse that constant?

Reply:
In principle I'm slightly in favour of a specific constant, possibly with the same value. Anyway, you have a better picture of the whole code, so if you think that the semantic is similar enough then I'm OK with reusing `MAX_BUFFER_LENGTH`.